src/test/java/io/strimzi/kafka/access/Base64Encoder.java

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.kafka.access;
+
+import java.nio.charset.StandardCharsets;
+import java.util.Base64;
+
+public class Base64Encoder {
+
+    private static final Base64.Encoder ENCODER = Base64.getEncoder();
+
+    public static String encodeToString(String data) {
```

Comment:
This should probably be `encodeUtf8ToString()` (or just `encodeUtf8() perhaps, since base64 by definition returns text), because it encodes the String as UTF-8 implicitly to obtain the binary data to be base64 encoded. But it's a test class so it's not a big deal.

Reply:
I think you're right that naming it `encodeUtf8` is more descriptive of what it's actually doing.